tests/test_packageAlerts.py

```diff
@@ -357,7 +357,9 @@ def testCreateCcdDataCutout(self):
         ccdData = packageAlerts.createCcdDataCutout(
             self.exposure,
             self.exposure.getWcs().getSkyOrigin(),
-            self.exposure.getPhotoCalib())
+            self.exposure.getBBox().getDimensions(),
+            self.exposure.getPhotoCalib(),
+            1234)
```

Comment:
Please define `1234` as a variable (something like `diaSourceId = 1234) and pass that in, instead of a number. Also for the rest of this file where you are passing in a source ID.